src/rua_paths.rs

```diff
@@ -21,13 +25,16 @@ pub struct RuaPaths {
 	global_checked_tars_dir: PathBuf,
 	/// Script used to wrap `makepkg` and related commands
 	pub wrapper_bwrap_script: PathBuf,
-	/// Script used to read `makepkg` config
-	pub makepkg_config_loader: PathBuf,
+	/// makepkg configuration for PKGEXT
+	pub makepkg_pkgext: String,
```

Comment:
This is the core of the PR. `PKGEXT` is made part of `RuaPaths`.

Reply:
Oh boy, RIP accurate name for `RuaPaths`, we hardly knew ye :P

Overall the PR looks fine :+1:, but we might want to look into making some
side-effects more obvious in the future, so a similar regression won't happen again.

Reply:
Yeah, I started adding comments to those methods now. Also, changed the method name from `RuaPaths::new()` to `RuaPaths::initialize_paths()` in last commit. It has more "verb" to it I guess.